### ISSUE TITLE
Moveit2 run_moveit_cpp demo failing, ros2_control dies.

### ISSUE THREAD

**Issue by lilustga** [ms-iot/ROSOnWindows#343]
**Describe the bug**
When the run_moveit_cpp demo is run, the ros2_control node dies and the demo fails.

This seems to be due to an error with the panda_controller. Output is similar to this: https://gist.github.com/dpiet/5dc59cdddf2cc9694563dea6d27eb403

**To Reproduce**
 * Download the Foxy Setup which includes the 09-02 repos
 * Open terminal window for new foxy repo
 * `ros2 launch run_moveit_cpp run_moveit_cpp.launch.py`
 * Look for ros2 control process dying in the terminal.

**Environment information:**
 - OS version: 21H2

**Comment by lilustga** [comment]
This was due to a version mismatch caused by some packages being migrated to `joint_state_broadcaster` but not others. With `moveit`, `moveit_resources`, `ros2_control` and `ros2_controller` all moved forward, the `run_moveit_cpp` demo plans successfully.

**Comment by lilustga** [comment]
fixed by https://github.com/ms-iot/ros-windows-build/pull/206

**Comment by VinHub-Gitcent** [comment]
I think this issue is again happening on the current build:

```
ros2 launch run_moveit_cpp run_moveit_cpp.launch.py
[INFO] [launch]: All log files can be found below C:\Users\ZOVNITZK\.ros\log\2023-11-03-18-54-43-995021-ZO05N005Y0-8104
[INFO] [launch]: Default logging verbosity is set to INFO
[INFO] [static_transform_publisher.EXE-1]: process started with pid [16356]
[INFO] [robot_state_publisher.EXE-2]: process started with pid [33608]
[INFO] [rviz2.EXE-3]: process started with pid [33388]
[INFO] [run_moveit_cpp.EXE-4]: process started with pid [2664]
[INFO] [ros2_control_node.EXE-5]: process started with pid [3756]
[INFO] [ros2 run controller_manager spawner.py panda_arm_controller-6]: process started with pid [25108]
[INFO] [ros2 run controller_manager spawner.py panda_hand_controller-7]: process started with pid [19476]
[INFO] [ros2 run controller_manager spawner.py joint_state_broadcaster-8]: process started with pid [11332]
[static_transform_publisher.EXE-1] [INFO] [1699034084.991222300] [static_transform_publisher]: Spinning until killed publishing transform from 'world' to 'panda_link0'
[robot_state_publisher.EXE-2] Parsing robot urdf xml string.
[robot_state_publisher.EXE-2] Link panda_link1 had 1 children
[robot_state_publisher.EXE-2] Link panda_link2 had 1 children
[robot_state_publisher.EXE-2] Link panda_link3 had 1 children
[robot_state_publisher.EXE-2] Link panda_link4 had 1 children
[robot_state_publisher.EXE-2] Link panda_link5 had 1 children
[robot_state_publisher.EXE-2] Link panda_link6 had 1 children
[robot_state_publisher.EXE-2] Link panda_link7 had 1 children
[robot_state_publisher.EXE-2] Link panda_link8 had 1 children
[robot_state_publisher.EXE-2] Link panda_hand had 2 children
[robot_state_publisher.EXE-2] Link panda_leftfinger had 0 children
[robot_state_publisher.EXE-2] Link panda_rightfinger had 0 children
[robot_state_publisher.EXE-2] [INFO] [1699034085.251946800] [robot_state_publisher]: got segment panda_hand
[robot_state_publisher.EXE-2] [INFO] [1699034085.252133200] [robot_state_publisher]: got segment panda_leftfinger
[robot_state_publisher.EXE-2] [INFO] [1699034085.252156500] [robot_state_publisher]: got segment panda_link0
[robot_state_publisher.EXE-2] [INFO] [1699034085.252170500] [robot_state_publisher]: got segment panda_link1
[robot_state_publisher.EXE-2] [INFO] [1699034085.252182200] [robot_state_publisher]: got segment panda_link2
[robot_state_publisher.EXE-2] [INFO] [1699034085.252193200] [robot_state_publisher]: got segment panda_link3
[robot_state_publisher.EXE-2] [INFO] [1699034085.252203700] [robot_state_publisher]: got segment panda_link4
[robot_state_publisher.EXE-2] [INFO] [1699034085.252215000] [robot_state_publisher]: got segment panda_link5
[robot_state_publisher.EXE-2] [INFO] [1699034085.252225600] [robot_state_publisher]: got segment panda_link6
[robot_state_publisher.EXE-2] [INFO] [1699034085.252236200] [robot_state_publisher]: got segment panda_link7
[robot_state_publisher.EXE-2] [INFO] [1699034085.252246600] [robot_state_publisher]: got segment panda_link8
[robot_state_publisher.EXE-2] [INFO] [1699034085.252257400] [robot_state_publisher]: got segment panda_rightfinger
[run_moveit_cpp.EXE-4] [INFO] [1699034085.869376900] [moveit_cpp_demo]: Initialize node
[ros2_control_node.EXE-5] [INFO] [1699034086.883297100] [controller_manager]: update rate is 100 Hz
[rviz2.EXE-3] [INFO] [1699034088.206472000] [rviz2]: Stereo is NOT SUPPORTED
[rviz2.EXE-3] [INFO] [1699034088.206900900] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2.EXE-3] [INFO] [1699034088.776208100] [rviz2]: Stereo is NOT SUPPORTED
[rviz2.EXE-3] Parsing robot urdf xml string.
[rviz2.EXE-3] [INFO] [1699034090.190592900] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.0021705 seconds
[rviz2.EXE-3] [INFO] [1699034090.190731200] [moveit_robot_model.robot_model]: Loading robot model 'panda'...
[ros2_control_node.EXE-5] [INFO] [1699034090.954021700] [controller_manager]: Loading controller 'joint_state_broadcaster'
[ros2_control_node.EXE-5] [INFO] [1699034091.076842700] [controller_manager]: Loading controller 'panda_arm_controller'
[ros2_control_node.EXE-5] [INFO] [1699034091.219425700] [controller_manager]: Loading controller 'panda_hand_controller'
[ros2_control_node.EXE-5] [ERROR] [1699034091.219573200] [controller_manager]: Available classes:
[ros2_control_node.EXE-5] [ERROR] [1699034091.219624400] [controller_manager]: diff_drive_controller/DiffDriveController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219652700] [controller_manager]: effort_controllers/JointGroupEffortController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219680300] [controller_manager]: force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster
[ros2_control_node.EXE-5] [ERROR] [1699034091.219704700] [controller_manager]: forward_command_controller/ForwardCommandController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219727200] [controller_manager]: imu_sensor_broadcaster/IMUSensorBroadcaster
[ros2_control_node.EXE-5] [ERROR] [1699034091.219749900] [controller_manager]: joint_state_broadcaster/JointStateBroadcaster
[ros2_control_node.EXE-5] [ERROR] [1699034091.219771900] [controller_manager]: joint_state_controller/JointStateController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219798100] [controller_manager]: joint_trajectory_controller/JointTrajectoryController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219820000] [controller_manager]: position_controllers/JointGroupPositionController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219841400] [controller_manager]: velocity_controllers/JointGroupVelocityController
[ros2_control_node.EXE-5] [ERROR] [1699034091.219866100] [controller_manager]: Loader for controller 'panda_hand_controller' not found.
[ERROR] [ros2 run controller_manager spawner.py panda_hand_controller-7]: process has died [pid 19476, exit code 1, cmd 'ros2 run controller_manager spawner.py panda_hand_controller'].
[ros2_control_node.EXE-5] [INFO] [1699034091.557211800] [controller_manager]: Configuring controller 'joint_state_broadcaster'
[run_moveit_cpp.EXE-4] [INFO] [1699034091.678667500] [moveit_cpp_demo]: Initialize MoveItCpp
[run_moveit_cpp.EXE-4] Parsing robot urdf xml string.
[run_moveit_cpp.EXE-4] [INFO] [1699034091.688692200] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.0020273 seconds
[run_moveit_cpp.EXE-4] [INFO] [1699034091.688830400] [moveit_robot_model.robot_model]: Loading robot model 'panda'...
[ros2_control_node.EXE-5] [INFO] [1699034091.701532900] [controller_manager]: Configuring controller 'panda_arm_controller'
[ros2_control_node.EXE-5] [INFO] [1699034091.701762400] [panda_arm_controller]: Command interfaces are [position] and and state interfaces are [position velocity].
[ros2_control_node.EXE-5] [INFO] [1699034091.703810100] [panda_arm_controller]: Controller state will be published at 50.00 Hz.
[ros2_control_node.EXE-5] [INFO] [1699034091.706065500] [panda_arm_controller]: Action status changes will be monitored at 20.00 Hz.
[INFO] [ros2 run controller_manager spawner.py joint_state_broadcaster-8]: process has finished cleanly [pid 11332]
[ros2 run controller_manager spawner.py joint_state_broadcaster-8] [INFO] [1699034091.366299600] [spawner_joint_state_broadcaster]: Loaded joint_state_broadcaster

[INFO] [ros2 run controller_manager spawner.py panda_arm_controller-6]: process has finished cleanly [pid 25108]
[ros2 run controller_manager spawner.py panda_arm_controller-6] [INFO] [1699034091.478452300] [spawner_panda_arm_controller]: Loaded panda_arm_controller

[rviz2.EXE-3] Parsing robot urdf xml string.
[rviz2.EXE-3] [INFO] [1699034092.108027100] [moveit_rdf_loader.rdf_loader]: Loaded robot model in 0.0010359 seconds
[rviz2.EXE-3] [INFO] [1699034092.108086300] [moveit_robot_model.robot_model]: Loading robot model 'panda'...
[run_moveit_cpp.EXE-4] Link panda_link1 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link2 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link3 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link4 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link5 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link6 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link7 had 1 children
[run_moveit_cpp.EXE-4] Link panda_link8 had 1 children
[run_moveit_cpp.EXE-4] Link panda_hand had 2 children
[run_moveit_cpp.EXE-4] Link panda_leftfinger had 0 children
[run_moveit_cpp.EXE-4] Link panda_rightfinger had 0 children
[rviz2.EXE-3] [WARN] [1699034092.190358300] [moveit_ros.robot_model_loader]: No kinematics plugins defined. Fill and load kinematics.yaml!
[run_moveit_cpp.EXE-4] [INFO] [1699034092.207688200] [moveit.ros_planning_interface.moveit_cpp]: Listening to '/joint_states' for joint states
[run_moveit_cpp.EXE-4] [INFO] [1699034092.209301500] [moveit_ros.current_state_monitor]: Listening to joint states on topic '/joint_states'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.210496000] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/moveit_cpp/planning_scene_monitor' for attached collision objects
[run_moveit_cpp.EXE-4] [INFO] [1699034092.211549100] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Publishing maintained planning scene on '/moveit_cpp/monitored_planning_scene'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.211867100] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
[run_moveit_cpp.EXE-4] [INFO] [1699034092.212549100] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/moveit_cpp/publish_planning_scene'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.212593500] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[run_moveit_cpp.EXE-4] [INFO] [1699034092.214912600] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to 'collision_object'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.216718400] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[run_moveit_cpp.EXE-4] [WARN] [1699034092.216838500] [moveit.ros.occupancy_map_monitor]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[run_moveit_cpp.EXE-4] [ERROR] [1699034092.216924300] [moveit.ros.occupancy_map_monitor]: No 3D sensor plugin(s) defined for octomap updates
[run_moveit_cpp.EXE-4] [INFO] [1699034092.221334800] [moveit.ros_planning_interface.moveit_cpp]: Loading planning pipeline 'ompl'
[rviz2.EXE-3] [INFO] [1699034092.297462100] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Starting planning scene monitor
[rviz2.EXE-3] [INFO] [1699034092.299521600] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Listening to '/moveit_cpp/monitored_planning_scene'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.717673100] [moveit.ros_planning.planning_pipeline]: Using planning interface 'OMPL'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881588800] [moveit_ros.add_time_optimal_parameterization]: Param 'ompl.path_tolerance' was not set. Using default value: 0.100000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881663200] [moveit_ros.add_time_optimal_parameterization]: Param 'ompl.resample_dt' was not set. Using default value: 0.100000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881685600] [moveit_ros.add_time_optimal_parameterization]: Param 'ompl.min_angle_change' was not set. Using default value: 0.001000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881750200] [moveit_ros.fix_workspace_bounds]: Param 'ompl.default_workspace_bounds' was not set. Using default value: 10.000000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881808400] [moveit_ros.fix_start_state_bounds]: Param 'ompl.start_state_max_bounds_error' was set to 0.050000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881834300] [moveit_ros.fix_start_state_bounds]: Param 'ompl.start_state_max_dt' was not set. Using default value: 0.500000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881889200] [moveit_ros.fix_start_state_collision]: Param 'ompl.start_state_max_dt' was not set. Using default value: 0.500000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881906900] [moveit_ros.fix_start_state_collision]: Param 'ompl.jiggle_fraction' was not set. Using default value: 0.020000
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881923600] [moveit_ros.fix_start_state_collision]: Param 'ompl.max_sampling_attempts' was not set. Using default value: 100
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881979400] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Add Time Optimal Parameterization'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.881995400] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Workspace Bounds'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.882010300] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Bounds'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.882024300] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State In Collision'
[run_moveit_cpp.EXE-4] [INFO] [1699034092.882038400] [moveit.ros_planning.planning_pipeline]: Using planning request adapter 'Fix Start State Path Constraints'
[run_moveit_cpp.EXE-4] [WARN] [1699034092.945291800] [rcl.logging_rosout]: Publisher already registered for provided node name. If this is due to multiple nodes with the same name then all logs for that logger name will go out over the existing publisher. As soon as any node with that name is destructed it will unregister the publisher, preventing any further logs for that name from being published on the rosout topic.
[run_moveit_cpp.EXE-4] [INFO] [1699034093.059924100] [moveit.plugins.moveit_simple_controller_manager]: Added FollowJointTrajectory controller for panda_arm_controller
[rviz2.EXE-3] [INFO] [1699034097.321484600] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Failed to call service get_planning_scene, have you launched move_group or called psm.providePlanningSceneService()?
[run_moveit_cpp.EXE-4] [WARN] [1699034098.065349200] [moveit.simple_controller_manager.gripper_controller_handle]: Waiting for panda_hand_controller/gripper_cmd to come up
[run_moveit_cpp.EXE-4] [WARN] [1699034103.066586200] [moveit.simple_controller_manager.gripper_controller_handle]: Waiting for panda_hand_controller/gripper_cmd to come up
[run_moveit_cpp.EXE-4] [ERROR] [1699034108.077498900] [moveit.simple_controller_manager.gripper_controller_handle]: Action client not connected: panda_hand_controller/gripper_cmd
[run_moveit_cpp.EXE-4] [INFO] [1699034108.080506900] [moveit.plugins.moveit_simple_controller_manager]: Returned 1 controllers in list
[run_moveit_cpp.EXE-4] [INFO] [1699034108.081922500] [moveit_ros.trajectory_execution_manager]: Trajectory execution is not managing controllers
[run_moveit_cpp.EXE-4] [INFO] [1699034108.083164500] [moveit_cpp_demo]: Initialize PlanningComponent
[run_moveit_cpp.EXE-4] [INFO] [1699034111.083886700] [moveit_cpp_demo]: Set goal
[run_moveit_cpp.EXE-4] [INFO] [1699034111.084034400] [moveit_cpp_demo]: Plan to goal
[run_moveit_cpp.EXE-4] Debug:   Starting goal sampling thread
[run_moveit_cpp.EXE-4] [INFO] [1699034111.085873100] [moveit.ompl_planning.model_based_planning_context]: Planner configuration 'panda_arm' will use planner 'geometric::RRTConnect'. Additional configuration parameters will be set when the planner is constructed.
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Debug:   Beginning sampling thread computation
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Adding goal state
[run_moveit_cpp.EXE-4] Debug:   Stopped goal sampling thread after 10 sampling attempts
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (2 start + 3 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.003841 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000073 seconds hybridizing 1 solution paths (attempted 0 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 4 states (2 start + 2 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.001944 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000047 seconds hybridizing 2 solution paths (attempted 1 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (2 start + 3 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.003932 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000044 seconds hybridizing 3 solution paths (attempted 2 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (2 start + 3 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.002880 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000048 seconds hybridizing 4 solution paths (attempted 3 connections between paths)
[run_moveit_cpp.EXE-4] Info:    ParallelPlan::solve(): Solution found by one or more threads in 0.008459 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (3 start + 2 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.003601 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000108 seconds hybridizing 6 solution paths (attempted 11 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (2 start + 3 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.003728 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000095 seconds hybridizing 7 solution paths (attempted 6 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (2 start + 3 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.003103 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000082 seconds hybridizing 8 solution paths (attempted 7 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 6 states (4 start + 2 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.005039 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000107 seconds hybridizing 9 solution paths (attempted 8 connections between paths)
[run_moveit_cpp.EXE-4] Info:    ParallelPlan::solve(): Solution found by one or more threads in 0.007663 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: starting planner panda_arm/panda_arm
[run_moveit_cpp.EXE-4] Debug:   panda_arm/panda_arm: Planner range detected to be 6.712660
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Starting planning with 1 states already in datastructure
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 4 states (2 start + 2 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.002616 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000202 seconds hybridizing 11 solution paths (attempted 23 connections between paths)
[run_moveit_cpp.EXE-4] Info:    panda_arm/panda_arm: Created 5 states (2 start + 3 goal)
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Solution found by panda_arm/panda_arm in 0.005189 seconds
[run_moveit_cpp.EXE-4] Debug:   ParallelPlan.solveMore: Spent 0.000112 seconds hybridizing 12 solution paths (attempted 11 connections between paths)
[run_moveit_cpp.EXE-4] Info:    ParallelPlan::solve(): Solution found by one or more threads in 0.006555 seconds
[run_moveit_cpp.EXE-4] Info:    SimpleSetup: Path simplification took 0.011913 seconds and changed from 3 to 3 states
[run_moveit_cpp.EXE-4] [INFO] [1699034111.129948600] [moveit_cpp_demo]: arm.execute()
[ros2_control_node.EXE-5] [INFO] [1699034111.143598400] [panda_arm_controller]: Received new action goal
[run_moveit_cpp.EXE-4] [INFO] [1699034111.130193000] [moveit_ros.trajectory_execution_manager]: Validating trajectory with allowed_start_tolerance 0.01
[run_moveit_cpp.EXE-4] [INFO] [1699034111.142968300] [moveit_ros.trajectory_execution_manager]: Starting trajectory execution ...
[run_moveit_cpp.EXE-4] [INFO] [1699034111.143144800] [moveit.simple_controller_manager.follow_joint_trajectory_controller_handle]: sending trajectory to panda_arm_controller
[run_moveit_cpp.EXE-4] [INFO] [1699034111.143949700] [moveit.simple_controller_manager.follow_joint_trajectory_controller_handle]: panda_arm_controller started execution
[run_moveit_cpp.EXE-4] [INFO] [1699034111.143997400] [moveit.simple_controller_manager.follow_joint_trajectory_controller_handle]: Goal request accepted!
[ros2_control_node.EXE-5] [INFO] [1699034111.143705900] [panda_arm_controller]: Accepted new action goal
[run_moveit_cpp.EXE-4] [WARN] [1699034111.184283000] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
[run_moveit_cpp.EXE-4] [WARN] [1699034112.208673500] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: The complete state of the robot is not yet known.  Missing virtual_joint
[ros2_control_node.EXE-5] [INFO] [1699034113.127075000] [panda_arm_controller]: Goal reached, success!
[run_moveit_cpp.EXE-4] [INFO] [1699034113.127706700] [moveit.simple_controller_manager.follow_joint_trajectory_controller_handle]: Controller panda_arm_controller successfully finished
[run_moveit_cpp.EXE-4] [INFO] [1699034113.181169100] [moveit_ros.trajectory_execution_manager]: Completed trajectory execution with status SUCCEEDED ...
[run_moveit_cpp.EXE-4] [INFO] [1699034113.181669300] [moveit.ros_planning_interface.planning_component]: Deleting PlanningComponent 'panda_arm'
```
